specs/Autofill.md

```diff
@@ -0,0 +1,110 @@
+# Background
+The WebView2 team has been asked for an API to allow end developers to set the autofill preferences.  We are exposing two of the autofill preferences that will allow enabling and disabling general and password autofill.  General autofill includes things like email addresses, shipping addresses, phone numbers, and names.  Password autofill includes things like usernames and passwords for login.  Password information is not included in the general autofill. 
+
+In this document we describe the updated API. We'd appreciate your feedback.
+
+
+# Description
+
+Autofill has three components
```

Comment:
(Noting for posterity that Merriam-Webster does consider [autofill](https://www.merriam-webster.com/dictionary/autofill) and [autosave](https://www.merriam-webster.com/dictionary/autosave) to be single words.)